official/vision/beta/projects/yolo/utils/preprocessing_ops.py

```diff
@@ -0,0 +1,177 @@
+import tensorflow as tf
```

Comment:
add tests.

Reply:
Fixed in 5d40ec4ec5fe258206aa887e06e2f90082c22d46